packages/core/redux/bitcoin/BitcoinMiddlewareService.js

```diff
@@ -18,6 +19,8 @@ const BTC_MAINNET_NODE = axios.create({ baseURL: 'https://middleware-bitcoin-mai
 const BTC_TESTNET_NODE = axios.create({ baseURL: 'https://middleware-bitcoin-testnet-rest.chronobank.io', timeout: HTTP_TIMEOUT })
 const BCC_MAINNET_NODE = axios.create({ baseURL: 'https://explorer.bitcoingold.org/insight-api', timeout: 10000 })
 const BCC_TESTNET_NODE = axios.create({ baseURL: 'https://tbcc.blockdozer.com/insight-api', timeout: 10000, withCredentials: false })
+const DASH_MAINNET_NODE = axios.create({ baseURL: 'https://insight.dash.siampm.com/api', timeout: 10000 })
+const DASH_TESTNET_NODE = axios.create({ baseURL: 'https://testnet-insight.dashevo.org/insight-api-dash', timeout: 10000, withCredentials: false })
```

Comment:
`withCredentials: false` required only if we have CORS errors in HTTPS request to a host.
Is it required here or just copy-pasted?

Reply:
It is just a copy paste of a long string (`withCredentials: false` was hidden on my screen)